redbox/models/settings.py

```diff
@@ -65,7 +65,7 @@ class Settings(BaseSettings):
     django_settings_module: str = "redbox_app.settings"
     debug: bool = True
     django_secret_key: str
-    django_log_level: str
+    django_log_level: Literal["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"] = "WARNING"
```

Comment:
https://docs.djangoproject.com/en/5.0/topics/logging/#loggers

Reply:
We should probably just remove the setting altogether, and make extras optional as per https://stackoverflow.com/a/77021677/1073696.

Reply:
[0d7a495](https://github.com/i-dot-ai/redbox-copilot/pull/341/commits/0d7a4950c92b7c4db9ac52e2d7354d31f39e14c7)